src/System.Management.Automation/engine/hostifaces/LocalPipeline.cs

```diff
@@ -1219,25 +1179,25 @@ protected override
     }
 
     /// <summary>
-    /// Helper class that holds the thread used to execute pipelines when CreateThreadOptions.ReuseThread is used
+    /// Helper class that holds the thread used to execute pipelines when CreateThreadOptions.ReuseThread is used.
     /// </summary>
     internal class PipelineThread : IDisposable
     {
         /// <summary>
-        /// Creates the worker thread and waits for it to be ready
+        /// Creates the worker thread and waits for it to be ready.
         /// </summary>
 #if CORECLR
         internal PipelineThread()
         {
-            _worker = new Thread(WorkerProc);
+            _worker = new Thread(WorkerProc, LocalPipeline.DefaultPipelineStackSize);
```

Comment:
nit: maybe use `DefaultPipelineStackSize` too, just like in rest of the places.

Reply:
I don't understand your request - use `DefaultPipelineStackSize` instead of `LocalPipeline.DefaultPipelineStackSize`? - but it is another class.

Reply:
Ah, never mind, I saw you changed the `DefaultPipelineStackSize` in the below method to `LocalPipeline.DefaultPipelineStackSize`.